xl_auth/templates/collections/view.html

```diff
@@ -0,0 +1,93 @@
+<!-- collections/inspect.html -->
+{% extends "layout.html" %}
+{% block content %}
+    <h1>{{ _('View Collection \'%(code)s\'', code=collection.code) }}</h1>
+    <br/>
+    <div class="panel panel-default">
+        <dl class="dl-horizontal">
+            <dt>{{ _('Active') }}:</dt>
+            <dd>{{ _('Yes') if collection.is_active else _('No') }}</dd>
+
+            <dt>{{ _('Replaces') }}:</dt>
+            <dd>
+                {% if collection.replaces %}
+                    <a title="{{ _('View') }}"
+                       href="{{ url_for('collection.view', collection_code=collection.replaces) }}">
+                        {{ collection.replaces }}
+                    </a>
+                {% else %}
+                    -
+                {% endif %}
+            </dd>
+
+            <dt>{{ _('Replaced By') }}:</dt>
+            <dd>
+                {% if collection.replaced_by %}
+                    <a title="{{ _('View') }}"
+                       href="{{ url_for('collection.view', collection_code=collection.replaced_by) }}">
+                        {{ collection.replaced_by }}
+                    </a>
+                {% else %}
+                    -
+                {% endif %}
+            </dd>
+
+            <dt>{{ _('Last Modified') }}:</dt>
+            <dd>
+                {{ collection.modified_at.strftime('%Y-%m-%dT%H:%M:%SZ') }}, {{  _('by') }}
+                <a href="{{ url_for('user.view', user_id=collection.modified_by.id) }}">
+                    {{ collection.modified_by.full_name }}
+                </a>
+            </dd>
+
+            <dt>{{ _('Created At') }}:</dt>
+            <dd>
+                {{ collection.created_at.strftime('%Y-%m-%dT%H:%M:%SZ') }}, {{  _('by') }}
+                <a href="{{ url_for('user.view', user_id=collection.created_by.id) }}">
+                    {{ collection.created_by.full_name }}
+                </a>
+            </dd>
+        </dl>
+    </div>
+    <div class="panel panel-default">
+        <div class="panel-heading">
+            <strong>{{ _('Permissions') }}</strong>
```

Comment:
@mxtthias Since we've discussed differentiating this based on attributes of the viewer, this should be changed to say e.g.
* [ ] ``_('Permissions')`` when viewing as superuser
* [ ] ``_('Permissions')`` when viewing as a user that has the ``cataloging_admin`` permission on the collection
* [ ] ``_('Cataloging Admin Permissions')`` when viewing as a user that does not have ``cataloging_admin`` permission on the collection

Reply:
Maybe we want to always show the cataloging admins? For users and superusers, it would make it easier to figure out who to contact when it comes to permissions. For cataloging admins, it might make it easier to keep the permissions in order (e.g. "Hm, person A still has admin rights here, even though they changed jobs last month. I should remove this permission.").